content/logs/log_collection/rsyslog.md

```diff
@@ -75,7 +75,9 @@ description: "Configure Rsyslog to gather logs from your host, containers & serv
     sudo apt-get install rsyslog-gnutls
     ```
 
-    Download the public key for TLS encryption of logs from [this link][1]. Save it to `/etc/ssl/certs/intake.logs.datadoghq.com.crt`
+    Download the public key for TLS encryption of logs from [this link][1]. Save it to `/etc/ssl/certs/intake.logs.datadoghq.com.crt`.  
+    
+    **Note**: On some systems, the full certificate chain may be required.  If so, please download using [this link][2].
```

Comment:
@le-he-hoo is there a pattern on systems incriminated, like all linux/windows system?

Reply:
I _believe_ it has mostly been Ubuntu with versions of Rsyslog >= 8.x

Reply:
🤔 well let's be fuzzy on this one then...